python/graphstorm/trainer/ep_trainer.py

```diff
@@ -144,7 +144,7 @@ def fit(self, train_loader, num_epochs,
                 rt_profiler.record('train_graph2GPU')
 
                 # TODO(zhengda) we don't support edge features for now.
-                loss = model(blocks, batch_graph, input_feats, None, lbl, input_nodes)
+                loss = model(blocks, batch_graph, input_feats, None, None, lbl, input_nodes)
```

Comment:
in our document, we should say that these edge features are not valid right now.

Reply:
or will we have a followup PR to fetch edge features?